UnicornEngine/video/source/Camera.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/Camera.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+Camera::Camera(glm::vec3 postion, glm::vec3 direction)
+    : m_aspect(0), m_camPosition(postion), m_upVector(glm::vec3(0.0f, -1.0f, 0.0f)), m_camDirection(direction), m_fov(45.0f), m_znear(0.1f), m_zfar(1000.0f), m_projectionType(ProjectionType::Perspective)
```

Comment:
`m_upVector` is already `glm::vec3`, could write just `m_upVector(0.0f, -1.0f, 0.0f)`